qiskit/quantum_info/states/stabilizerstate.py

```diff
@@ -386,7 +386,12 @@ def probabilities(self, qargs: None | list = None, decimals: None | int = None)
 
         return probs
 
-    def probabilities_dict(self, qargs: None | list = None, decimals: None | int = None) -> dict:
+    def probabilities_dict_from_bitstring(
+        self,
+        outcome_bitstring: str,
+        qargs: None | list = None,
+        decimals: None | int = None,
+    ) -> dict[str, float]:
         """Return the subsystem measurement probability dictionary.
 
         Measurement probabilities are with respect to measurement in the
```

Comment:
could you please emphasize in the docstring what is the difference between `probabilities_dict_from_bitstring` and `probabilities_dict` ? it seems that both functions have the same docstring, which is a bit confusing.